linters/badtime/main.go

```diff
@@ -28,19 +28,21 @@ import (
 	"go/token"
 	"go/types"
 	"log"
-	"path"
 	"strings"
 
 	"golang.org/x/tools/go/loader"
 
 	"github.com/kisielk/gotool"
 )
 
-type callbackFunc func(loc token.Position, keyStr, valStr string)
+type mapCallbackFunc func(loc token.Position, keyStr, valStr string)
```

Comment:
looks like these have the same signature, would it be a better abstraction to share the type? 

something like:
```
type validationCallbackFn func(loc token.Position, keyStr, valStr string)
```

Reply:
see why you've kept them separate, disregard.